### ISSUE TITLE
Add an option to specify the first day of a week

### ISSUE THREAD

**Issue by pyrog** [opening-hours/opening_hours.js#326]
In France for example, the week start on monday

**Comment by ypid** [comment]
I agree very much. In Germany it is also Monday. I just have not gotten around to make it configurable and provide better defaults.
